src/Analysis/GWASResults/Views/Input/AttritionTable/AttrtitionTable.jsx

```diff
@@ -0,0 +1,105 @@
+  const { selectedRowData } = useContext(SharedContext);
+  const { name, uid } = selectedRowData;
+  const { data, status } = useQuery(
+    [`getDataForWorkflowArtifact${name}`, name, uid, 'attrition_json_index'],
+    () => getDataForWorkflowArtifact(name, uid, 'attrition_json_index'),
+    queryConfig,
+  );
+
+  if (status === 'error') {
+    return (
+      <LoadingErrorMessage
+        data-testid='loading-error-message'
+        message='Error getting attrition table data'
+      />
+    );
+  }
+
+  if (status === 'loading') {
+    return (
+      <div className='spinner-container' data-testid='spinner'>
+        <Spin />
+      </div>
+    );
+  }
+
+  if (!data || data.length === 0 || data.error) {
+    return (
+      <LoadingErrorMessage message='Issue Loading Data for Attrition Table' />
+    );
+  }
+
+  return (
+    <section data-testid='attrition-table' className='attrition-table'>
+      <div className='attrition-table'>
+        <Collapse
+          defaultActiveKey={['1']}
+          onClick={(event) => event.stopPropagation()}
+        >
+          <Panel key='1' header='Attrition Table'>
+            <table>
+              <thead>
+                <tr>
+                  <th className='attrition-table--leftpad attrition-table--w15'>
+                    Type
+                  </th>
+                  <th className='attrition-table--w15'>Name</th>
+                  <th className='attrition-table--rightborder attrition-table--w5'>
+                    Size
+                  </th>
+                  <th className='attrition-table--w15 attrition-table--leftpad'>
+                    Non-Hispanic Black
+                  </th>
+                  <th className='attrition-table--w15'>Non-Hispanic Asian</th>
+                  <th className='attrition-table--w15'>Non-Hispanic White</th>
+                  <th className='attrition-table--w15'>Hispanic</th>
+                </tr>
+              </thead>
+              <tbody>
+                {data[0].rows.map((row, index) => (
+                  <tr key={index}>
+                    <td>{row?.type || <h3>❌</h3>}</td>
+                    <td>{row?.name || <h3>❌</h3>}</td>
+                    <td className='attrition-table--rightborder'>
+                      {row?.size || <h3>❌</h3>}
+                    </td>
+                    <td>
+                      {row?.concept_breakdown[0]
+                        ?.persons_in_cohort_with_value || <h3>❌</h3>}
+                    </td>
+                    <td>
+                      {row?.concept_breakdown[1]
+                        ?.persons_in_cohort_with_value || <h3>❌</h3>}
+                    </td>
+                    <td>
+                      {row?.concept_breakdown[2]
+                        ?.persons_in_cohort_with_value || <h3>❌</h3>}
+                    </td>
+                    <td>
+                      {row?.concept_breakdown[3]
+                        ?.persons_in_cohort_with_value || <h3>❌</h3>}
```

Comment:
please consider writing a function to get the `concept_breakdown` that matches a given group name, e.g. :
instead of 
```js
row.concept_breakdown[3]
```
write something like:
```js
getBreakDownForGroup('Hispanic', row.concept_breakdown)
```
where this function returns the row.concept_breakdown item that matches the given group name ('Hispanic' in this case).

In this way, if the order of the columns changes in the JSON data, this code still works.

Reply:
This would be similar to what you did here: https://github.com/uc-cdis/data-portal/commit/276e4ae9a73dcd6c920d59ed09ccd6867f68cacd#diff-b3bb99e8959e9147cb9cf9dd63dd6c478e45968657599dfd578bb4c159ea1e2dR74-R82

Reply:
This has been updated to use a getBreakDownForGroup function as described: 
```
 const getBreakDownForGroup = (groupName, conceptBreakdownArray) => {
    const matchingObject = conceptBreakdownArray.find(
      (obj) => obj.concept_value_name === groupName,
    );
    return matchingObject?.persons_in_cohort_with_value || <h3>❌</h3>;
  };

```